frontend/public/components/storage/create-pvc.tsx

```diff
@@ -164,6 +175,15 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
       },
     };
 
+    // should set block only for RBD + RWX
+    if (
+      storageClass &&
+      _.endsWith(provisioner, cephRBDProvisionerSuffix) &&
+      accessMode === 'ReadWriteMany'
+    ) {
+      obj.spec.volumeMode = 'Block';
+    }
+
```

Comment:
Can we move this inside this ?
https://github.com/openshift/console/blob/b8fe1f1517877aa752c89448c52498b72d452a6b/frontend/public/components/storage/create-pvc.tsx#L193-L195